### ISSUE TITLE
t.p.c.Interface shouldn't warn when a user inherits from another interface

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11082]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)| @jyknight reported|
|-|-|
|Trac ID|trac#1599|
|Type|enhancement|
|Created|2006-03-21 21:10:29Z|

Interface.__init__ emits a warning when non-twisted packages inherit from it, suggesting using zope.interface.Interface instead. However, if the user is inheriting from a twisted interface that in turn inherits from t.p.c.Interface, they don't have that choice. So it shouldn't emit the warning.

<details><summary>Searchable metadata</summary>

```
trac-id__1599 1599
type__enhancement enhancement
reporter__jknight jknight
priority__normal normal
milestone__Core_2_5 Core-2.5
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1142975429000000 1142975429000000
changetime__1151771310000000 1151771310000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

Milestone Twisted-2.3 deleted

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

Sorry, deferred again.

**Comment by twisted-trac** [comment]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)|@jyknight set status to `closed`|
|-|-|

Fixed by implication by fix for [#11150](https://github.com/twisted/twisted/issues/11150).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
